app/_includes/footer.html

```diff
@@ -17,22 +17,30 @@
         </ul>
       </li>
       <li>
-        <a href="/about/people">Members</a>
+        <a href="/members/">Members</a>
```

Comment:
no its part of scope, because social media links were a part of footer but it was requested that it have its own section. The members link was moved to create space for social media links but then it was added back, since social media links had their own space